app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,8 @@
+# Controller to handle mediations for admins
+class AdminController < ApplicationController
+
+  def index
+    @requests = Request.all.group_by(&:origin).sort
+    authorize! :index, @requests
```

Comment:
@jkeck Tried `authorize! :manage, Request` before `@requests` assignment. Didn't work. Ended up leaving the line as it is